build.yaml

```diff
@@ -1397,26 +1362,25 @@ steps:
               --doctest-glob='*.rst' \
               --ignore=docs/conf.py \
               --ignore=docs/doctest_write_data.py
```

Comment:
@daniel-goldstein first guess: some shenanigans with the python path. The stack traces point at the `/io/hail/python/hail` directory:
```
/io/hail/python/hail/conftest.py:37: 
```
Instead of `cd`'ing can you just `python3 -m pytest hail/python/hail/ ...`? You should be able to replicate locally by installing (non editable) and then cd'ing and running pytest.